gpo/crud.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import Session
+
+from . import models, schemas
+
+
+def get_letters(db: Session, skip: int = 0, limit: int = 1000):
```

Comment:
Took me a few mins going through sqlalchemy docs to find out this returns a list of `models.Letter`. Perhaps a type-hint to make this easier in the future?

Also appears to be using the "1.x" style querying from sqlalchemy instead of 2.x. Is this intentional? https://docs.sqlalchemy.org/en/14/orm/session_basics.html#querying-2-0-style